config/betterquesting/DefaultQuests/Quests/StoringandTransf-AAAAAAAAAAAAAAAAAAAAIQ==/LowVoltageTransf-AAAAAAAAAAAAAAAAAAAE8g==.json

```diff
@@ -31,7 +31,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "It is possible to run MV machines while still in LV - build a transformer and use 4 power sources 32 eu/t to power one 128 eu/t machine.\n\nIt transform§rs power up or down, depending on the mode, which you can switch with a soft mallet. Step-up means 4A LV in, 1A MV out, while step-down means the reverse.\n\n§4Make sure§r you understand the following if you prefer your base not being a crater:\nA step-down mode is the default one. In this mode, the large dot is the input. In a step-up mode, it\u0027s the output. §4Do not§r mix it up. LV machines that receive MV power will explode.\n\nIf the transformer is connected to power, or was in the past, then §4do not§r change the mode with stuff connected. It\u0027s better to break and place it again to get rid of the stored power rather than rotate it with the wrench and blow things up. Make sure to also disconnect the nearby wires with a wire cutter.\n\nAs a reminder, machines don\u0027t blow up if they are given more amps than they can consume. They blow up from higher voltage tier."
+      "desc:8": "It is possible to run MV machines while still in LV - build a transformer and use 4 power sources 32 eu/t to power one 128 eu/t machine.\n\nIt transform§rs power up or down, depending on the mode, which you can switch with a soft mallet. Step-up means 4A LV in, 1A MV out, while step-down means the reverse.\n\nMake sure you understand the following if you prefer your base not being a crater:\nA step-down mode is the default one. In this mode, the large dot is the input. In a step-up mode, it\u0027s the output. Do not mix it up. LV machines that receive MV power will explode.\n\nIf the transformer is connected to power, or was in the past, then do not change the mode with stuff connected. It\u0027s better to break and place it again to get rid of the stored power rather than rotate it with the wrench and blow things up. Make sure to also disconnect the nearby wires with a wire cutter.\n\nAs a reminder, machines don\u0027t blow up if they are given more amps than they can consume. They blow up from higher voltage tier."
```

Comment:
It is one of the quests that had formatting spread around that I removed because it didn't conform to any standard. I want to redo the descriptions to emphasize key points like this, but consistently across all quests.